static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -59,7 +58,10 @@ export function InviteMissingMembersModal({
 
   const api = useApi();
 
-  if (!memberInvites || !organization.access.includes('org:write')) {
+  if (
+    (memberInvites && memberInvites.length === 0) ||
```

Comment:
do you need the `memberInvites &&` part here? Seems like given the types, memberInvites will not be null/undefined.

Reply:
i think you're right